src/components/Landing.js

```diff
@@ -110,77 +107,75 @@ export default class Landing extends Component {
       let result = results[index];
       //require additional processing of result data
       if (item.processFunction && this[item.processFunction]) {
-        result = this[item.processFunction](results[index][item.dataKey]);
+        try {
+          result = this[item.processFunction](results[index] ? results[index][item.dataKey] : null, item.dataKey);
+        } catch(e) {
+          console.log(`Error processing data result via processing function ${item.processFunction}: ${e}`);
+        }
       }
       dataSet[item.dataKey] = result ? result: null;
 
     });
     return dataSet;
   }
 
-  processOccupationData (result) {
-    if (!result || !result.length) return null;
+  processEndPoint(endpoint) {
+    if (!endpoint) return "";
+    return (endpoint)
+    .replace('{env.REACT_APP_CONF_API_URL}', process.env.REACT_APP_CONF_API_URL)
+    .replace('{env.PUBLIC_URL}', process.env.PUBLIC_URL);
```

Comment:
Just curious- why not use backticks (`) for string formatting vs multiple `.replace()`?